packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -202,11 +202,12 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 
         this.messageBuffer.push(message);
 
+        this.emit("submitOp", message);
```

Comment:
Changes looks good. But please test them without this change! (Including running through CI loop). Many clients will not have that change for many weeks after the rest of changes deploys.

Reply:
Sure - will run npm run ci:test  before proceeding.